### DISCUSSION TITLE
Fabric 1.19

### DISCUSSION THREAD

**Discussion by giorov** [maruohon/minihud#251] [Q&A]
Hey, sorry, just a newb about minihud and litematica. I run my Minecraft client on fabric mods on 1.19. Is there no way to use litematica and minihud on these? 
I guess not from what I see. 
But I don't see how anyone could even get them to work with 1.18 or 1.19 all. Does liteloader 1.12.2 support 1.19 or something?

**Comment by giorov** [comment]
I found the answer to the question, sort of... https://www.curseforge.com/minecraft/mc-mods/litematica

**Comment by maruohon** [comment]
Yes you can use the community ports linked on the top of the page for now. I have thus far released malilib and Item Scroller "officially", the other mods still need a bit more work.

Note however that you can't mix the "official" and community versions, mostly just due to how the mod version numbers are set up. So use the community versions of all of them until all the mods you use have their official releases out.
